### PR TITLE
Date time data generator and some improvements

### PR DESCRIPTION
**Pull request by prakashmrec**
This PR adds Date and Time data generator, removes methods for default values and adds convenience methods for Boolean and Computer Address data generators

### CODE REVIEW

File: src/main/java/com/smartbear/readyapi/client/teststeps/datasource/datagen/DateAndTimeDataGeneratorBuilder.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        return this;
+    }
+
+    /**
+     * Used for generating sequential values, ignored if generation mode is Random. Default value: 1 day (24 * 60 * 60 * 1000 milliseconds)
+     *
+     * @param milliSeconds increment value in milliseconds
+     * @return DateAndTimeDataGeneratorBuilder
+     */
+    public DateAndTimeDataGeneratorBuilder incrementWith(long milliSeconds) {
```
Comment (olensmar) [inline src/main/java/com/smartbear/readyapi/client/teststeps/datasource/datagen/DateAndTimeDataGeneratorBuilder.java]: I think it should be incrementBy...

File: src/test/java/com/smartbear/readyapi/client/DateAndTimeDataGenDataSourceTest.java [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+    @Test
+    public void buildsRecipeWithDataSourceTestStepWithDateAndTimeDataGenDataSourceWithProvidedStartAndEndDates() throws Exception {
+        Date startDate = new ISO8601DateFormat().parse("2016-01-01T17:26:20Z");
+        Date endDate = new ISO8601DateFormat().parse("2020-11-21T15:16:40Z");
+        TestRecipe recipe = newTestRecipe()
+                .addStep(dataGenDataSource()
+                        .withProperty(
+                                sequentialDateAndTimeTypeProperty("property1")
+                                        .withStartDate(startDate)
```
Comment (olensmar) [inline src/test/java/com/smartbear/readyapi/client/DateAndTimeDataGenDataSourceTest.java]: I guess you could do "startingAt" and "endingAt" instead!?

**Comment by olensmar** [comment]
looks good - very minor comments from my side :-)

**Comment by prakashmrec** [comment]
Fixed all the comments. Thanks.

**Comment by olensmar** [comment]
thank you!

**Pull request merged**